packages/loctool/lib/XliffMerge.js

```diff
@@ -31,21 +32,34 @@ var logger = log4js.getLogger("loctool.lib.XliffMerge");
  */
 var XliffMerge = function XliffMerge(settings) {
     if (!settings) return;
-
-    var target = new Xliff({
-        path: settings.outfile,
-        version: settings.xliffVersion,
-        style: settings.xliffStyle
-    });
-
+    if (settings.xliffStyle == 'webOS') {
```

Comment:
Implement a `XliffFactory()` and call it here. This should look at the xliffStyle and return the right type of xliff instance.

Reply:
I updated it.